package.json

```diff
@@ -58,14 +58,14 @@
   },
   "dependencies": {
     "@babel/polyfill": "^7.0.0",
-    "@playkit-js/playkit-js": "0.80.10",
+    "@playkit-js/playkit-js": "https://github.com/kaltura/playkit-js.git#FEC-12549",
     "@playkit-js/playkit-js-dash": "1.31.4",
     "@playkit-js/playkit-js-hls": "1.31.2",
     "@playkit-js/playkit-js-ui": "0.73.0",
     "@types/preact-i18n": "^2.3.1",
     "hls.js": "1.2.8",
     "intersection-observer": "^0.12.0",
-    "playkit-js-providers": "https://github.com/kaltura/playkit-js-providers.git#v2.37.0",
```

Comment:
you're directing to the branch ? does this work ? just remember to change it if you delete the branch

Reply:
I have to, in order for the flow type check to pass (some type imported from provider package)